src/sentry/static/sentry/app/views/stream/actions.jsx

```diff
@@ -3,12 +3,246 @@ import Reflux from 'reflux';
 import ApiMixin from '../../mixins/apiMixin';
 import ActionLink from './actionLink';
 import DropdownLink from '../../components/dropdownLink';
+import Duration from '../../components/duration';
 import IndicatorStore from '../../stores/indicatorStore';
 import MenuItem from '../../components/menuItem';
 import PureRenderMixin from 'react-addons-pure-render-mixin';
 import SelectedGroupStore from '../../stores/selectedGroupStore';
 import {t, tn} from '../../locale';
 
+import CustomIgnoreCountModal from '../../components/customIgnoreCountModal';
+import CustomIgnoreDurationModal from '../../components/customIgnoreDurationModal';
+
+const IgnoreActions = React.createClass({
```

Comment:
this is VERY similar to the existing IgnoreActions. It adds in the required props for ActionLink, and uses those instead of anchors, but otherwise is mostly copypasta